GitUI/UserControls/RevisionGrid/Graph/LaneInfoProvider.cs

```diff
@@ -11,6 +11,8 @@ namespace GitUI.UserControls.RevisionGrid.Graph
     internal sealed class LaneInfoProvider
     {
         private static readonly TranslationString NoInfoText = new TranslationString("Sorry, this commit seems to be not loaded.");
+        private static readonly TranslationString MergedWithText = new TranslationString("merged with");
```

Comment:
The string should probably be "merged with {0}" to support right-to-left languages - but there are many issues anyway...
Or is it even required to parse correctly? (regexp on line 85?)

Reply:
Good point, changed it.
Parsing works independently. Though if Git's merge messages can be internationalized, then the regexp must be adapted, too.